plugin/edit-form/edit.form.panel.view.js

```diff
@@ -0,0 +1,65 @@
+import {PluginView} from '../plugin.view';
+import {isUndefined} from '@grid/core/utility';
+import {Command} from '@grid/core/command';
+import {RowEditor} from '@grid/core/edit/edit.row.editor';
+import {Event} from '@grid/core/infrastructure';
+
+export class EditFormPanelView extends PluginView {
+	constructor(model, context) {
+		super(model);
+		
+		this.model = model;
```

Comment:
I don't see the reason to do this explicitly

Reply:
good point